infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/SenderBufferManager.java

```diff
@@ -53,8 +52,7 @@
     /*
      * If there is a timeout for a message, should generate an error or not
      */
-    private boolean errorOnMsgTimeout = true;
-
+    private boolean errorOnMsgTimeout;
```

Comment:
default value will be false which can change the behavior.. is it intentional?

Reply:
the static code analysis asks to remove this redundant. It is initialized in the constructor.